dev/com.ibm.websphere.appserver.features/visibility/public/cdi-4.0/io.openliberty.cdi-4.0.feature

```diff
@@ -65,7 +69,7 @@ Subsystem-Name: Jakarta Contexts and Dependency Injection 4.0
  io.openliberty.cdi.4.0.internal.weld, \
  com.ibm.ws.cdi.interfaces.jakarta, \
  io.openliberty.cdi.spi; location:="dev/spi/ibm/,lib/"
--jars=io.openliberty.cdi.4.0.thirdparty; location:="dev/api/third-party/,lib/"; mavenCoordinates="org.jboss.weld:weld-osgi-bundle:5.0.0.Alpha2"
+-jars=io.openliberty.cdi.4.0.thirdparty; location:="dev/api/third-party/,lib/"; mavenCoordinates="org.jboss.weld:weld-osgi-bundle:5.0.0"
```

Comment:
This maven coordinate says 5.0.0 instead of 5.0.0.Beta1?

Reply:
>that jar really isn't the same as the bundle you would find at those maven co-ordinates!

I think that's potentially a problem. IIRC the liberty maven and gradle plugins use this to add the right API dependencies when the user specifies dependencies on particular features.